requirements/test.txt

```diff
@@ -4,72 +4,68 @@
 #
 #    make upgrade
 #
-astroid==1.6.6            # via pylint, pylint-celery
-atomicwrites==1.3.0       # via pytest
+appdirs==1.4.3            # via virtualenv
+astroid==2.3.3            # via pylint, pylint-celery
 attrs==19.3.0             # via pytest
-backports.functools-lru-cache==1.6.1  # via astroid, isort, pylint
 certifi==2019.11.28
 chardet==3.0.4
 click-log==0.3.2          # via edx-lint
 click==7.0                # via click-log, edx-lint
-configparser==4.0.2       # via importlib-metadata, pylint
-contextlib2==0.6.0.post1  # via importlib-metadata
 coverage==4.5.4
 ddt==1.2.2
-django-waffle==0.18.0
+distlib==0.3.0            # via virtualenv
+django-waffle==0.20.0
 djangorestframework-jwt==1.11.0
-djangorestframework==3.9.4
-edx-django-utils==2.0.2
+edx-django-utils==3.0
 edx-lint==1.4.1
 edx-opaque-keys==2.0.1
-enum34==1.1.6             # via astroid
 factory-boy==2.12.0
-faker==2.0.4              # via factory-boy
-funcsigs==1.0.2           # via mock, pytest
+faker==4.0.1              # via factory-boy
+filelock==3.0.12          # via virtualenv
 future==0.18.2
-futures==3.1.1 ; python_version == "2.7"
+futures==3.1.1
```

Comment:
I thought this package was both broken and unnecessary under Python 3; can we just remove it from `tests.in`?